tests/components/matter/test_binary_sensor.py

```diff
@@ -344,3 +345,30 @@ async def test_water_valve(
     state = hass.states.get("binary_sensor.valve_valve_leaking")
     assert state
     assert state.state == "on"
+
+
+@pytest.mark.parametrize("node_fixture", ["thermostat"])
+async def test_thermostat_occupancy(
+    hass: HomeAssistant,
+    matter_client: MagicMock,
+    matter_node: MatterNode,
+) -> None:
+    """Test thermostat occupancy."""
+    state = hass.states.get("binary_sensor.longan_link_hvac_occupancy")
+    assert state
+    assert state.state == "on"
+
+    # Test Occupancy attribute change
+    occupancy_attribute = clusters.Thermostat.Attributes.Occupancy
+
+    set_node_attribute(
+        matter_node,
+        1,
+        occupancy_attribute.cluster_id,
+        occupancy_attribute.attribute_id,
```

Comment:
note: all other binary sensor tests seem to hardcode these values, but this is completely fine (/better) IMO.